resources/fetchersManager/registry_test.go

```diff
@@ -15,17 +15,16 @@
 // specific language governing permissions and limitations
 // under the License.
 
```

Comment:
Same as the other test files

Reply:
not sure I can revert everything here but I'll try